### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by ddiaz2380** [TiagoSilvaPereira/vemto-issues#160] [Ideas & Suggestions]
It would be great if they could add what language I want my application to be generated. Since I use Spanish and then I have to translate all the buttons and others by hand.

**Comment by Loukili-Rachid** [comment]
i agree with you it would be a great feature, i,m using Arabic i face the same case always

**Comment by joselleida** [comment]
incluso, trabajar con varios idiomas a la vez
